test/legacy/multipart.test.js

```diff
@@ -139,6 +139,78 @@ test('should call finished when both files are pumped', { skip: process.platform
   })
 })
 
+test('should call finished if one of the streams closes prematurely', { skip: process.platform === 'win32' }, function (t) {
+  t.plan(5)
+
+  const fastify = Fastify({
+    logger: {
+      level: 'debug'
+    }
```

Comment:
remove logging as we don't want the tests to output anything

Reply:
Done